package.json

```diff
@@ -107,7 +107,7 @@
     "lint-staged": "8.2.1",
     "mri": "1.1.4",
     "postcss": "7.0.17",
-    "postcss-custom-properties": "8.0.11",
+    "postcss-custom-properties": "9.0.1",
```

Comment:
It mainly updated an internal dep and dropped node@6 support.

Reply:
Be careful, we had problems with this update

Reply:
I had a lot of problems with this version 

https://github.com/postcss/postcss-custom-properties/issues/186

Reply:
Okay so best to not update this one I presume.